luatest/server.lua

```diff
@@ -208,6 +208,10 @@ function Server:http_request(method, path, options)
     return response
 end
 
+function Server:eval(command)
```

Comment:
Arguments and options should be passed as well. See doc [here](https://www.tarantool.io/en/doc/latest/reference/reference_lua/net_box/#lua-function.conn.eval).